tinyevals/dataset.py

```diff
@@ -1,11 +1,21 @@
+import numpy as np
 from datasets import load_dataset
 from tqdm.auto import tqdm
 
 def load_clean_dataset(split: str, tokenized: bool = False) -> list[str]:
     # checking just startswith, because you can include slice like "train[:1000]"
     assert split.startswith("train") or split.startswith("validation")
-    hf_ds = load_dataset(f"jbrinkma/tinystories-v2-clean{'-tokenized' if tokenized else ''}")
+    hf_ds = load_dataset(f"jbrinkma/tinystories-v2-clean{'-tokenized' if tokenized else ''}", split=split)
     dataset = []
-    for sample_txt in tqdm(hf_ds["tokens" if tokenized else "text"]):
-        dataset.append(sample_txt)
+    for sample in tqdm(hf_ds["tokens" if tokenized else "story"]):
+        dataset.append(sample)
     return dataset
+
+def token_map(tokenized_dataset: list[str]) -> dict[int, list[tuple[int, int]]]:
```

Comment:
shouldn't it be
```suggestion
def token_map(tokenized_dataset: list[list[int]]) -> dict[int, list[tuple[int, int]]]:
```
?